deploy/olm-catalog/service-telemetry-operator/manifests/service-telemetry-operator.clusterserviceversion.yaml

```diff
@@ -211,6 +191,18 @@ spec:
     spec:
       clusterPermissions:
       - rules:
+        - apiGroups:
+          - apiextensions.k8s.io
+          resourceNames:
+          - servicetelemetrys.infra.watch
```

Comment:
I actually think I can drop this to be honest. I was using some previous work as the kick off, and I have a feeling this was here to allow reading the CRD as part of some debugging. I'll go through everything again and see if this can be removed. If so, then I'll try to put back the other `resources` list in the CSV, although I'm not really sure if _that_ is necessary.

I noticed that it had ServiceMonitors for the old CRD path when using Prometheus Operator (coreos.com) vs the COO one (rhobs). Since everything seems to work, as far as I can tell this is just informational, and we can probably just leave it in the simplified form?

Reply:
Actually I think this whole block might have been for debugging purposes, so there is a good chance this whole thing will go away.